README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
+   + I have recently come around to some genre permeability with fantasy
+      + If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
+  + My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
+   + here is the link to watch the best Trek while stuck inside. [You'll thank me](https://www.netflix.com/title/70158330?source=35) 
```

Comment:
your humor came across with the funny use of "You'll thank me - is was a creative way to add a link title, instead of just inserting the link.